src/OpenTelemetry.Exporter.Geneva/Internal/ConnectionStringBuilder.cs

```diff
@@ -137,7 +137,7 @@ public string ParseUnixDomainSocketPath()
         try
         {
             var endpoint = new Uri(this.Endpoint);
-            return endpoint.AbsolutePath;
+            return endpoint.AbsolutePath.Replace('@','\0');
```

Comment:
Only replace when `@` is the first character?

Reply:
Is that required to merge this?